elasticsearch_django/apps.py

```diff
@@ -90,8 +90,9 @@ def _on_model_delete(sender, **kwargs):
 def _update_search_index(instance, action, force=False):
     """Process generic search index update actions."""
     # this allows us to turn off sync temporarily - e.g. when doing bulk updates
-    if not settings.get_setting('auto_sync'):
-        logger.debug("SEARCH_AUTO_SYNC disabled, ignoring update.")
+    model_name = "{}.{}".format(instance._meta.app_label, instance._meta.model_name)
+    if not settings.get_setting('auto_sync') or model_name in settings.get_setting('never_auto_sync'):
```

Comment:
Minor change on this - just to make the logging less ambiguous:

```suggestion
    if not settings.get_setting('auto_sync'):
        logger.debug("SEARCH_AUTO_SYNC disabled, ignoring update.")
        return

    if model_name in settings.get_setting('never_auto_sync'):
        logger.debug("Model (%s) listed in never_auto_sync, ignoring update.", model_name)
        return
```